com.adobe.Flash-Player-Projector.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "com.adobe.Flash-Player-Projector",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flashplayer",
+    "separate-locales": false,
+    "tags": ["proprietary"],
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Play sounds */
+        "--socket=pulseaudio"
+    ],
+    "modules" : [
+        {
+            "name" : "flashplayer",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -D -t /app/bin/ apply_extra",
+                "install flashplayer /app/bin",
+                "install -Dm644 -t /app/share/icons/ com.adobe.Flash-Player-Projector.svg",
+                "install -Dm644 -t /app/share/applications/ com.adobe.Flash-Player-Projector.desktop",
+                "install -Dm644 -t /app/share/metainfo/ com.adobe.Flash-Player-Projector.appdata.xml"
+            ],
+            "sources" : [
+                {
+                    "type" : "script",
+                    "dest-filename" : "flashplayer",
+                    "commands": [
+                       "exec /app/extra/bin/flashplayer.bin $@"
+                    ]
+                },
+                {
+                   "type" : "script",
+                   "dest-filename": "apply_extra",
+                   "commands": [
+                       "echo Linked from https://www.adobe.com/support/flashplayer/debug_downloads.html",
```

Comment:
If this is supposed to be a comment, notice that manifest in YAML (with proper comments) are supported!

Reply:
So we do with json with C-style comments or prepending a field with underscore.

Reply:
Except that the C-style comments didn't work:
```
                   "commands": [
                       /* Linked from https://www.adobe.com/support/flashplayer/debug_downloads.html" */
                       "tar xvzf flash_player_sa_linux.x86_64.tar.gz",
                       "rm -f flash_player_sa_linux.x86_64.tar.gz"
                   ]
```

```
Can't parse 'com.adobe.Flash-Player-Projector.json': <data>:37:67: Parse error: unexpected identifier `support', expected value
```

Because the C-style comments choke on the `:`.

It also warns about unknown fields:
```
                   "_linked_from": "https://www.adobe.com/support/flashplayer/debug_downloads.html",
```
yields:
```
(flatpak-builder:4060): flatpak-builder-WARNING **: 12:14:58.269: Unknown property _linked_from for type BuilderSourceScript
```

So I'll leave this as-is.